ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/config/server_error_default.yaml

```diff
@@ -0,0 +1,7 @@
+defaultMessageTemplate: "  No worries, though! You can still send us your tracked data manually. Here's what to do:
+            
+            Copy this link: <i>%s</i>
+            Open the folder using the link.
+            Once you're in the folder, please send us all its files.
```

Comment:
For now, we create several extra folders, e.g. with test photos from camera devices. But all logged data is stored in the root of the plugin folder. Should we create a folder for all tracked data and put all csv files into this folder? In this case, we can put the link to this folder directly and avoid sending some extra personal data such the photos